index.html

```diff
@@ -1,414 +1,358 @@
+                        </p>
+                        <p class="show-on-lg hide-on-sm">
+                          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea </p>
+                    </div>
+
+                    <div class="modal-button-conatiner">
+                        <div class="modal-technologies">
+                            <ul class="list grid-list">
+                                <li><div class="tag">html</div></li>
+                                <li><div class="tag">css</div></li>
+                                <li><div class="tag">javascript</div></li>
+                                <li class="hide-on-sm show-on-lg"><div class="tag">github</div></li>
+                                <li class="hide-on-sm show-on-lg"><div class="tag">ruby</div></li>
+                                <li class="hide-on-sm show-on-lg"><div class="tag">Bootstraps</div></li>
+                            </ul>
+                        </div>
+                        <div class="modal-buttons flex align-center justify-center">
+                            <a href="https://1gordon.github.io/my-portfolio/
+                            " class="button-style modal-btn">See Live <img class="button-icons .button-icons-color" src="./images/see-live-icon.svg" alt="see live icon"></a>
+                            <a href="https://github.com/1GORDON/my-portfolio
+                            " class="button-style modal-btn">See Source<img class="button-icons .button-icons-color" src="./images/github.svg" alt="github icon"></a>
+                        </div>
+                    </div>
+                </div>
             </div>
-        </section>
-    </section>
-</section>
+        </div>
+    </div>
+
+    <!-- project section -->
+    <section class="projects"></section>
```

Comment:
- [ ] [OPTIONAL] Kindly add more padding for your tech stack labels or fix the content according to the size of your tech stack name. 😉 
- [ ] [OPTIONAL] Make it responsive, you can use min-width CSS attribute to fix it 👍 (check the second image below) 

![image](https://user-images.githubusercontent.com/60164239/132025608-2196c8b1-9d7a-4c34-8264-3be7fce4732f.png)

![image](https://user-images.githubusercontent.com/60164239/132026056-85841ecd-f1a0-489c-be6d-e11d7f8f3f3d.png)